lib/membrane_http_adaptive_stream/hls.ex

```diff
@@ -4,23 +4,20 @@ defmodule Membrane.HTTPAdaptiveStream.HLS do
 
   Currently supports up to one audio and video stream.
   """
-
   @behaviour Membrane.HTTPAdaptiveStream.Manifest
 
-  alias Membrane.HTTPAdaptiveStream.Manifest
-  alias Membrane.HTTPAdaptiveStream.Manifest.Track
+  use Ratio
+
+  alias Membrane.HTTPAdaptiveStream.{BandwidthCalculator, Manifest}
   alias Membrane.Time
 
   @version 7
 
-  @av_manifest """
-  #EXTM3U
-  #EXT-X-VERSION:#{@version}
-  #EXT-X-INDEPENDENT-SEGMENTS
-  #EXT-X-MEDIA:TYPE=AUDIO,NAME="a",GROUP-ID="a",AUTOSELECT=YES,DEFAULT=YES,URI="audio.m3u8"
-  #EXT-X-STREAM-INF:BANDWIDTH=2560000,CODECS="avc1.42e00a",AUDIO="a"
-  video.m3u8
-  """
+  @master_playlist_header "#EXTM3U\n#EXT-X-VERSION:#{@version}\n#EXT-X-INDEPENDENT-SEGMENTS"
```

Comment:
Wy don't you use the same syntax as was used before?
sth like:
```
 @master_playlist_header """
#EXTM3U
#EXT-X-VERSION:#{@version}
#EXT-X-INDEPENDENT-SEGMENTS
"""
```

Reply:
I would prefer that syntax too

Reply:
Previous syntax implies '\n' character at the end of the string. Without '\n' character I can put master_playlist_header into list that will be be passed to Enum.join('\n"). Otherwise I would have to concatenate it separately which makes another part of the code less readable. I am strongly against reverting this change.

Reply:
Done.